x-pack/legacy/plugins/ml/public/jobs/jobs_list/jobs.js

```diff
@@ -13,6 +15,14 @@ import { JobsListView } from './components/jobs_list_view';
 export const JobsPage = (props) => (
   <Fragment>
     <NavigationMenu tabId="jobs" />
+    <EuiTitle style={{ padding: '16px 16px 0 16px' }}>
+      <h1>
+        <FormattedMessage
+          id="xpack.ml.jobsList.title"
+          defaultMessage="Anomaly Detection"
```

Comment:
I think this should be `Anomaly detection jobs` for consistency with `Analytics jobs`

Reply:
Fixed in https://github.com/elastic/kibana/pull/49464/commits/03817fe1465f591ce2d28dbb2e104636049e37f8